src/sparsezoo/model/utils.py

```diff
@@ -599,9 +599,8 @@ def _copy_and_overwrite(from_path, to_path, func):
 
 def include_file_download_url(files: List[Dict]):
     for file in files:
-        file["url"] = get_file_download_url(
-            model_id=file["model_id"], file_name=file["display_name"]
-        )
+        file["url"] = get_file_download_url(file["download_url"])
+        del file["download_url"]
```

Comment:
why delete?

Reply:
Not required, just trying to maintain the existing file dict interface of sorts. I'd think having both a `url` and `download_url` property would confuse anyone looking at the dict and trying to make sense of things